app/views/layouts/application.slim

```diff
@@ -122,11 +121,12 @@ html
           = yield(:footer)
           .row
             .col-md-6
-              - if User.joins(:patreon).where('patreon_users.pledge_cents > 0').any?
+              - patreons = User.includes(:patreon, :twitch).joins(:patreon).where('patreon_users.pledge_cents >= 200')
```

Comment:
When talking about users who contribute to the Patreon, use `patrons` over `patreons`